.github/workflows/build.yml

```diff
@@ -151,9 +151,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [18, 16]
-        backend: ['geth', 'ganache']
-        mode: ['http', 'ws']
+        node: [18]
+        backend: ['ganache']
```

Comment:
the ganache will be deprecated what do you think about keeping only geth?

Reply:
Sure! But there is another issue to replace Ganache with Hardhat. So, this will be replaced rather than deleted. So, I kept it for smoother merging the code later with the other task that is in progress...